TF_Tooling_Web/src/Cursors/SvgCursor.ts

```diff
@@ -78,8 +79,10 @@ export class SVGCursor extends TouchlessCursor {
 
         this.ringSizeMultiplier = ringSizeMultiplier;
 
-        ConnectionManager.instance.addEventListener('HandFound', this.ShowCursor.bind(this));
-        ConnectionManager.instance.addEventListener('HandsLost', this.HideCursor.bind(this));
+        TouchFree.RegisterEventCallback('HandFound', this.ShowCursor.bind(this));
+        TouchFree.RegisterEventCallback('HandsLost', this.HideCursor.bind(this));
+        TouchFree.RegisterEventCallback('HandEntered', this.ShowCursor.bind(this));
+        TouchFree.RegisterEventCallback('HandExited', this.HideCursor.bind(this));
```

Comment:
Could you add these to the DotCursor too?

Reply:
Added!